Source/Core/Common/Common.vcxproj

```diff
@@ -175,6 +176,7 @@
     <ClCompile Include="Hash.cpp" />
     <ClCompile Include="IniFile.cpp" />
     <ClCompile Include="JitRegister.cpp" />
+    <ClCompile Include="LdrWatcher.cpp" />
```

Comment:
Hm...I doubt it worked ever...cmake also needs to set linker flag /include:<symbol_name> , it would have needed to include ucrtFreadWorkaround , and now enableCompatPatches. I'm not sure how to do that.